zigen-project/zen
# Implement config & wallpaper again

## Context
#152 #160

## Summary
Re-implemented the following two features that existed in the previous versions.
- Wallpaper (system background image)
- toml config file

## How to check behavior

1. Run `zen-desktop -s weston-terminal` and confirm that the default background image is shown.
2. Create the following config file on `$HOME/.config/zen-desktop/config.toml` and confirm that the image is used as the wallpaper.

```toml
[wallpaper]
filepath = "/path/to/your/image/file" 
```
